From 6e96520aad6308fe94da2b14bf4be2dcb028d4d1 Mon Sep 17 00:00:00 2001
From: moshe <moshe@yahoo.com>
Date: Fri, 14 Jul 2017 10:29:17 +0530
Subject: [PATCH 1/2] extra added

---
 main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/main.c b/main.c
index cf97c87..4bbd495 100644
--- a/main.c
+++ b/main.c
@@ -4,4 +4,5 @@ void main()
 
 printf("welcome to git");
   printf("added printf");
+printf("extar addede");
 }
-- 
1.9.1

